conn.go

```diff
@@ -731,11 +731,13 @@ func (c *conn) connWriter() {
 			// send close
 			cls := &frames.PerformClose{}
 			debug(1, "TX (connWriter): %s", cls)
-			_ = c.writeFrame(frames.Frame{
+			err = c.writeFrame(frames.Frame{
```

Comment:
It'll be interesting if we start seeing this error bubble up. I can't imagine it's too harmful, but still.

Reply:
Agreed, as it might indicate we have some race during shutdown.

Reply:
I'm undoing this change for now as I believe that `writeFrame()` can race with closing the underlying connection which would lead to an infinite loop.

Reply:
Opened https://github.com/Azure/go-amqp/issues/155 to track this.